Library/Formula/app-engine-java-sdk.rb

```diff
@@ -2,8 +2,8 @@
 
 class AppEngineJavaSdk < Formula
   homepage "https://developers.google.com/appengine/docs/java/gettingstarted/introduction"
-  url "https://storage.googleapis.com/appengine-sdks/featured/appengine-java-sdk-1.9.19.zip"
-  sha256 "22a761ea50af1d1a0e1f698cc3fa91765391902a633ca886ea1a6426c3ae1ccb"
+  url "https://storage.googleapis.com/appengine-sdks/featured/appengine-java-sdk-1.9.21.zip"
+  sha1 "a405c32aa72d90c404fff0cab59624bf44d9c43d"
```

Comment:
We're not going to downgrade the `sha` checksum here. Google themselves have essentially [rained on](http://googleonlinesecurity.blogspot.com/2014/09/gradually-sunsetting-sha-1.html) SHA1. Keep it at `sha256`.